source/_integrations/modbus.markdown

```diff
@@ -274,7 +274,7 @@ modbus:
     baudrate: 9600
     bytesize: 8
     method: rtu
-    parity: E
```

Comment:
Why is 'N' better than 'E' ?

my line is setup with Even parity.

These are just examples.

Reply:
Ok, its up to you. 
But even in your documentaiont it is written ( N - by default). 
Can you say what modbus devices do you use with E parity?

Reply:
Most devices can configure the parity.